wezterm-gui/src/termwindow/render.rs

```diff
@@ -1144,6 +1163,51 @@ impl super::TermWindow {
         Ok(())
     }
 
+    pub fn populate_box_drawing_quad(
+        &self,
+        box_drawing: BoxDrawingKey,
+        gl_state: &RenderState,
+        quads: &mut MappedQuads,
+        cell_idx: usize,
+        params: &RenderScreenLineOpenGLParams,
+        hsv: Option<config::HsbTransform>,
+        cursor_shape: Option<CursorShape>,
+        glyph_color: LinearRgba,
+        underline_color: LinearRgba,
+        bg_color: LinearRgba,
+        white_space: TextureRect,
+    ) -> anyhow::Result<()> {
+        let sprite = gl_state
+            .glyph_cache
+            .borrow_mut()
+            .cached_box_drawing(box_drawing)?
```

Comment:
This function is 99% duplicated from `populate_block_quad`, with the exception being this call here, so there's plenty of room to DRY up this logic.